lib/compat/wordpress-6.5/rest-api.php

```diff
@@ -144,3 +144,16 @@ function _gutenberg_register_wp_templates_additional_fields() {
 }
 
 add_action( 'rest_api_init', '_gutenberg_register_wp_templates_additional_fields' );
+/**
+ * Registers the Block Rederer REST API routes.
+ */
+function gutenberg_register_block_rederer_routes() {
```

Comment:
If this is experimental and not targetted for 6.5 then this code should be in the `experiments` directory.